.github/workflows/security.yml

```
@@ -0,0 +1,122 @@
+    - name: Check licenses
+      run: |
+        go-licenses report ./... --template licenses.tpl > licenses.md || true
+        echo "## Dependency Licenses" >> $GITHUB_STEP_SUMMARY
+        echo "\`\`\`" >> $GITHUB_STEP_SUMMARY
+        go-licenses report ./... 2>/dev/null || echo "Some dependencies may not have detectable licenses" >> $GITHUB_STEP_SUMMARY
+        echo "\`\`\`" >> $GITHUB_STEP_SUMMARY
+
+  container-scan:
+    name: Container Security (if Dockerfile exists)
+    runs-on: ubuntu-latest
+    if: hashFiles('Dockerfile*') != ''
```

actionlint (1.7.7)

calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)